plat/kvm/Config.uk

```diff
@@ -101,6 +101,8 @@ config KVM_VMM_FIRECRACKER
 	bool "Firecracker"
 	select KVM_BOOT_PROTO_LXBOOT
 	select HAVE_MMIO
+	select HAVE_PS2
+	imply LIBUKPS2_KBD
```

Comment:
Although the kbd driver already depends on `ARCH_X86_64`, I find it looks like it's universally enabled here, so perhaps change to `imply LIBUKPS2_KBD if ARCH_X86_64`? We also follow that pattern for GIC.